pkg/report/datafeeds/evm/decode.go

```diff
@@ -0,0 +1,38 @@
+package evm
```

Comment:
Why is this in an evm package? This is a chainlink-evm repo

Reply:
this was a mistake leftover from my early prototyping